mobile/lib/modules/asset_viewer/ui/remote_photo_view.dart

```diff
@@ -16,13 +16,17 @@ class _RemotePhotoViewState extends State<RemotePhotoView> {
   Widget build(BuildContext context) {
     bool allowMoving = _status == _RemoteImageStatus.full;
 
-    return PhotoView(
-      imageProvider: _imageProvider,
-      minScale: PhotoViewComputedScale.contained,
-      maxScale: allowMoving ? 1.0 : PhotoViewComputedScale.contained,
-      enablePanAlways: true,
-      scaleStateChangedCallback: _scaleStateChanged,
-      onScaleEnd: _onScaleListener,
+    return SizedBox(
+      height: MediaQuery.of(context).size.height,
```

Comment:
Why do we need to wrap the `SizedBox` here?

Reply:
Flutter throws an error for having undefined/infinite height and width.

Reply:
Never mind, I guess I accidentally fixed it after I encountered the error haha